src/views/Auth.vue

```diff
@@ -1,11 +1,15 @@
 <template>
-  <div>
+  <div class="container">
+    <!-- TODO: MAKE TABS INSTEAD OF BUTTONS -->
```

Comment:
Can we copy the styling from https://dash.jackblanc.com/auth for toggling between sign in/sign up here? With the text below the two inputs.

Reply:
Sure I'll get on it asap